src/Uno.UI/Microsoft/UI/Xaml/Controls/ColorPicker/ColorPickerSlider.cs

```diff
@@ -0,0 +1,286 @@
+		{
+			if (m_toolTip is ToolTip toolTip)
+			{
+				toolTip.Content = this.GetToolTipString();
+
+				// ToolTip doesn't currently provide any way to re-run its placement logic if its placement target moves,
+				// so toggling IsEnabled induces it to do that without incurring any visual glitches.
+				toolTip.IsEnabled = false;
+				toolTip.IsEnabled = true;
+			}
+
+			DependencyObject currentObject = this;
+			ColorPicker owningColorPicker = null;
+
+			while (currentObject != null && (owningColorPicker = currentObject as ColorPicker) == null)
+			{
+				currentObject = VisualTreeHelper.GetParent(currentObject);
+			}
+
+			if (owningColorPicker != null)
+			{
+				Color oldColor = owningColorPicker.Color;
+				Hsv hsv = ColorConversion.RgbToHsv(ColorConversion.RgbFromColor(oldColor));
+				hsv.V = args.NewValue / 100.0;
+				Color newColor = ColorConversion.ColorFromRgba(ColorConversion.HsvToRgb(hsv));
+
+				ColorPickerSliderAutomationPeer peer = FrameworkElementAutomationPeer.FromElement(this) as ColorPickerSliderAutomationPeer;
+				peer.RaisePropertyChangedEvent(oldColor, newColor, (int)(Math.Round(args.OldValue)), (int)(Math.Round(args.NewValue)));
+			}
+		}
+
+		private ColorPicker GetParentColorPicker()
+		{
+			ColorPicker parentColorPicker = null;
+			DependencyObject currentObject = this;
+
+			while (currentObject != null && !(parentColorPicker is ColorPicker))
+			{
+				currentObject = VisualTreeHelper.GetParent(currentObject);
+			}
+
+			return parentColorPicker;
+		}
+
+		private string GetToolTipString()
+		{
+			uint sliderValue = (uint)(Math.Round(this.Value));
+
+			if (this.ColorChannel == ColorPickerHsvChannel.Alpha)
+			{
+				return string.Format(
+					ResourceAccessor.GetLocalizedStringResource(ResourceAccessor.SR_ToolTipStringAlphaSlider),
+					sliderValue);
+			}
+			else
+			{
+				ColorPicker parentColorPicker = GetParentColorPicker();
+				if (parentColorPicker != null && DownlevelHelper.ToDisplayNameExists())
+				{
+					Hsv currentHsv = parentColorPicker.GetCurrentHsv();
+					string localizedString;
+
+					switch (this.ColorChannel)
+					{
+						case ColorPickerHsvChannel.Hue:
+							currentHsv.H = this.Value;
+							localizedString = ResourceAccessor.GetLocalizedStringResource(ResourceAccessor.SR_ToolTipStringHueSliderWithColorName);
+							break;
+
+						case ColorPickerHsvChannel.Saturation:
+							localizedString = ResourceAccessor.GetLocalizedStringResource(ResourceAccessor.SR_ToolTipStringSaturationSliderWithColorName);
+							currentHsv.S = this.Value / 100;
+							break;
+
+						case ColorPickerHsvChannel.Value:
+							localizedString = ResourceAccessor.GetLocalizedStringResource(ResourceAccessor.SR_ToolTipStringValueSliderWithColorName);
+							currentHsv.V = this.Value / 100;
+							break;
+						default:
+							throw new Exception("Invalid ColorPickerHsvChannel."); // Uno Doc: 'throw winrt::hresult_error(E_FAIL);'
```

Comment:
Codacy found an issue: ['System.Exception' should not be thrown by user code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6121259)